### DISCUSSION TITLE
Integrate Auto Update for Auth Desktop

### DISCUSSION THREAD

**Discussion by Jishnuraj9** [ente-io/ente#2081] [Enhancements]
Add an automatic update feature to the Ente Auth app, removing the need for users to download updates from GitHub manually.

**Comment by blackhat840** [comment]
I would like to second this request.  With the updates so frequent, it would be nice if Ente Auth could auto update the desktop application on Windows.

**Comment by SixelAlexiS90** [comment]
As a new Auth user I quote this!

**Comment by kendofriendo** [comment]
Why in the year of our lord 2024 do we need to download and install a new version on every release. It's absurd.

**Comment by Damariobros** [comment]
It should be able to be turned on and off, though. If someone wants to defer, they should be able to update when they are ready, and if, for whatever reason, someone wants or needs an older version, they should be able to download it, install it, and not be immediately greeted with it terminating and updating itself.

**Comment by AdrianoCahete** [comment]
Why not release on Microsoft Store and winget, on windows?

**Reply by CullenStClair** [reply]
I have had success with using winget for updates

**Comment by samschultzponsys** [comment]
I would also really appreciate in program updating method

**Comment by dancingdog** [comment]
Updating on Linux installed a new version each time leaving me with a proliferation of entries which I couldn't tell apart. In-place update would be great.

**Comment by Hardtarget24** [comment]
how are we in 2026 and this still hasn't been done yet?
